dev-client/src/screens/LocationScreens/components/soilId/SoilIdMatchesSection.tsx

```diff
@@ -65,26 +66,23 @@ export const SoilIdMatchesSection = ({
           <TopSoilMatchesInfoContent isSite={isSite} />
         </InfoButton>
       </Row>
+      <RestrictByConnectivity offline={true}>
+        <OfflineMessageBox message={t('site.soil_id.matches.offline')} />
+      </RestrictByConnectivity>
       <MatchTilesOrMessage siteId={siteId} coords={coords} />
```

Comment:
Could rename from `MatchTilesOrMessage` to just `MatchTiles` or `MatchResults` or `MatchesArea` or something, since the message is no longer part of this component

Reply:
Good idea.